CppCoreGuidelines.md

```diff
@@ -4724,7 +4724,7 @@ These operations disagree about copy semantics. This will lead to confusion and
 
 ## <a name="SS-dtor"></a>C.dtor: Destructors
 
-"Does this class need a destructor?" is a surprisingly powerful design question.
```

Comment:
We agree with changing this, but think "insightful" would be better, would you update to that please?

Reply:
replaced as suggested